### PR TITLE
Improving API with to_matplotlib(...) and related

### PR DESCRIPTION
**Pull request by stared**
Adding a chainable way to specify outputs, without too many import lines, see #113 by @vfdev-5.

- [X] `reset_outputs`, `to_matplotlib`, `to_extrema_printer`
- [x] checking if it actually works
- [x] tests
- [x] following with all other outputs

### CODE REVIEW

**Comment by Bartolo1024** [comment]
In the case of python 3.5 or 3.6 we should use string annotations to specify self type:
```
def to_matplotlib(self, *args, **kwargs) -> 'PlotLosses':
```
instead of:
```
def to_matplotlib(self, *args, **kwargs) -> PlotLosses:
```
[stack](https://stackoverflow.com/questions/33533148/how-do-i-specify-that-the-return-type-of-a-method-is-the-same-as-the-class-itsel)
[python < 3.7 docs](https://www.python.org/dev/peps/pep-0484/#forward-references)
[python 3.7+ docs](https://www.python.org/dev/peps/pep-0563/)

File: livelossplot/__init__.py [hunk 1]
```diff
@@ -9,6 +9,12 @@
 from . import outputs
 from .version import __version__
 
+
+"""livelossplot
+
+Main repository: <https://github.com/stared/livelossplot>
```
Comment (Bartolo1024) [inline livelossplot/__init__.py:15]: I have noticed out how to place a Readme.md in the documentation. 
You have to include:
```
.. include:: ../README.md
```
in this docstring.
I have checked it in my [test repository](https://github.com/Bartolo1024/test/blob/master/explib/__init__.py) and it [works](https://bartolo1024.github.io/test) :sunglasses:

File: livelossplot/inputs/__init__.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ def PlotLossesPoutyne(**kwargs):
 def PlotLossesIgnite(**kwargs):
     """PlotLosses callback for Poutyne, a library for PyTorch.
     Args:
-        kwargs: key-arguments which are passed to PlotLosses
+        **kwargs: key-arguments which are passed to PlotLosses
 
     Notes:
         Requires pytorch-ignite to be installed, https://github.com/pytorch/ignite.
```
Comment (Bartolo1024) [inline livelossplot/inputs/__init__.py:43]: Brackets should be also there

File: livelossplot/inputs/__init__.py [hunk 3]
```diff
@@ -37,7 +37,7 @@ def PlotLossesPoutyne(**kwargs):
 def PlotLossesIgnite(**kwargs):
     """PlotLosses callback for Poutyne, a library for PyTorch.
```
Comment (vfdev-5) [inline livelossplot/inputs/__init__.py:38]:
```suggestion
    """PlotLosses callback for PyTorch-Ignite, a library for PyTorch.
```
A typo fix :)

**Pull request merged**